packages/miniflare/src/workers/d1/database.worker.ts

```diff
@@ -113,15 +132,20 @@ export class D1DatabaseObject extends MiniflareDurableObject {
 		return changes;
 	}
 
-	#query = (query: D1Query): D1SuccessResponse => {
+	#query = (format: D1ResultsFormat, query: D1Query): D1SuccessResponse => {
 		const beforeTime = performance.now();
 
 		const beforeSize = this.state.storage.sql.databaseSize;
 		const beforeChanges = this.#changes();
 
 		const params = convertParams(query.params ?? []);
 		const cursor = this.db.prepare(query.sql)(...params);
-		const results = convertResults(all(cursor));
```

Comment:
Looks like you'd need to remove the import for `all`

Reply:
Done!